assessment.sql

```diff
@@ -5,24 +5,64 @@ CREATE DATABASE "sql-crud-assessment";
 
 -- Question 1. Create a table called `licensees` to hold data in columns surname,given_name,gender,height,weight,dob,eye_color,hair_color.  See `data/licensees.csv` for example data.  Don't forget that all tables should have an id column as a unique identifier for a row.
 
--- your code here
+CREATE TABLE licensees (
+  id SERIAL PRIMARY KEY
+  surname CHARACTER VARYING,
+  given_name CHARACTER VARYING,
+  gender CHARACTER VARYING,
+  height integer,
+  weight integer,
+  dob CHARACTER VARYING,
+  eye_color CHARACTER VARYING,
+  hair_color CHARACTER VARYING
+);
```

Comment:
Hell yah, I think this is the first one I got from you :D

On Wed, Nov 4, 2015 at 3:34 PM, Saad Rhoulam <notifications@github.com>
wrote:

> In assessment.sql
> <https://github.com/ga-wdi-boston/sql-crud-assessment/pull/29#discussion_r43935525>
> :
>
> > @@ -5,24 +5,64 @@ CREATE DATABASE "sql-crud-assessment";
> >
> >  -- Question 1. Create a table called `licensees` to hold data in columns surname,given_name,gender,height,weight,dob,eye_color,hair_color.  See `data/licensees.csv` for example data.  Don't forget that all tables should have an id column as a unique identifier for a row.
> >
> > --- your code here
> > +CREATE TABLE licensees (
> > +  id SERIAL PRIMARY KEY
> > +  surname CHARACTER VARYING,
> > +  given_name CHARACTER VARYING,
> > +  gender CHARACTER VARYING,
> > +  height integer,
> > +  weight integer,
> > +  dob CHARACTER VARYING,
> > +  eye_color CHARACTER VARYING,
> > +  hair_color CHARACTER VARYING
> > +);
>
> Aside from the missing comma, this is good. 1
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ga-wdi-boston/sql-crud-assessment/pull/29/files#r43935525>
> .
>



-- 
Regards,
Jeff Meyer

857-206-3572